src/stores/todoStore.js

```
@@ -3,22 +3,42 @@ import { defineStore } from 'pinia'
+      } else if (state.filter === 'completed') {
+        filtered = filtered.filter(todo => todo.completed);
+      }
+      
+      // Filter by category
+      if (state.categoryFilter !== 'all') {
+        filtered = filtered.filter(todo => todo.category === state.categoryFilter);
+      }
+      
+      // Filter by tag
+      if (state.tagFilter !== 'all') {
+        filtered = filtered.filter(todo => todo.tags && todo.tags.includes(state.tagFilter));
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)